superset/db_engine_specs/base.py

```diff
@@ -1304,6 +1304,28 @@ def get_column_spec(
             )
         return None
 
+    @classmethod
+    def get_cancel_query_payload(cls, cursor: Any, query: Query) -> Any:
+        """
+        Returns None if query can not be cancelled.
```

Comment:
Could you provide a more descriptive description of what this function does, especially for the non-base case?

Reply:
addressed by ef8436b783a2baac74322ff2b98c0189edc332e6